src/main/java/in/ashwanthkumar/gocd/slack/GoNotificationMessage.java

```diff
@@ -188,7 +195,7 @@ public void tryToFixStageResult(Rules rules)
                                              Integer.parseInt(pipeline.stage.counter));
         if (previous == null || StringUtils.isEmpty(previous.result)) {
             LOG.info("Couldn't find any previous run of " +
-                     pipeline.name + "/" + pipeline.counter + "/" +
+                     pipeline.name + "/" + pipeline.counter + "/" + pipeline.label + "/" +
```

Comment:
I think it would be great if we can have the pipeline after the name than after the counter like:
 
> pipeline.name + "(" + pipeline.label + ")" + "/" + pipeline.counter + "/" +

Reply:
Changes are done as per your suggestion.